### PR TITLE
Image assets

### LINKED ISSUE #4: Assets
cargar imagenes y fonts

### PR DESCRIPTION
**Pull request by holden-caulfield**
Fixes #4

### CODE REVIEW

File: app/src/Screens/Help/index.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 import React, { Component } from 'react'
-import { Text, View } from 'react-native'
+import { Text, View, Image } from 'react-native'
 
 import PanicButton from 'app/Components/PanicButton'
+import images from 'app/Theme/images'
 import styles from './style'
 
+const remote = 'https://s15.postimg.org/tw2qkvmcb/400px.png'
```
Comment (rodrigouroz) [inline app/src/Screens/Help/index.js]: Can we host this locally?

File: app/src/Theme/images.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const images = {
+  background: require('../Images/textura.png')
```
Comment (rodrigouroz) [inline app/src/Theme/images.js]: Let's use names in english.

File: app/src/Screens/Help/index.js (hunk 1 above)
Comment (holden-caulfield) [reply app/src/Screens/Help/index.js]: sorry... that line is supposed to go was a test. The actual image is hosted locally. I will remove that line

File: app/src/Theme/images.js (hunk 2 above)
Comment (gonrial) [reply app/src/Theme/images.js]: uppercase, lowercase? it there a rule for this?

Comment (holden-caulfield) [reply app/src/Theme/images.js]: Good point on the english, I'll make the change.
Rule that I'm following for casing: all "packages" (i.e. folders) + modules that export React Components => uppercase
Files and every other  module => lowercase

**Pull request merged**